### PR TITLE
Feat: Add timer, CBR , Security levels ACC-406

### PR DESCRIPTION
**Pull request by MarcinR**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/ACC-406" title="ACC-406" target="_blank">SubtaskACC-406</a>  [iOS] Add timer, "CONSTANT BIT RATE" and "VARIABLE BIT RATE" , Security level
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [x] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability

### CODE REVIEW

File: Wire-iOS/Sources/Helpers/SemanticColors.swift [hunk 1]
```diff
@@ -64,6 +64,7 @@ public enum SemanticColors {
         static let textErrorDefault = UIColor(light: Asset.Colors.red500Light, dark: Asset.Colors.red500Dark)
         static let textPasswordRulesCheck = UIColor(light: Asset.Colors.gray80, dark: Asset.Colors.gray20)
         static let textTabBar = UIColor(light: Asset.Colors.gray70, dark: Asset.Colors.gray60)
+        static let textSecurityLevelClassified = UIColor(light: Asset.Colors.blue500Light, dark: Asset.Colors.blue500Dark)
```
Comment (KaterinaWire) [inline Wire-iOS/Sources/Helpers/SemanticColors.swift:67]: I think it should be an accent color, ask Anette

File: Wire-iOS/Sources/Helpers/SemanticColors.swift [hunk 2]
```diff
@@ -128,6 +129,9 @@ public enum SemanticColors {
         static let backgroundPurple = UIColor(light: Asset.Colors.purple100Light, dark: Asset.Colors.purple900Dark)
         static let backgroundTurqoise = UIColor(light: Asset.Colors.turquoise100Light, dark: Asset.Colors.turquoise900Dark)
         static let backgroundOverlay = UIColor(light: Asset.Colors.black, dark: Asset.Colors.black)
+        static let backgroundSecurityLevelUnclassified = UIColor(light: Asset.Colors.blue500Light, dark: Asset.Colors.blue500Dark)
+        static let backgroundSecurityLevelClassified = UIColor(light: Asset.Colors.blue50Light, dark: Asset.Colors.blue900Light)
+        static let borderSecurityLevelClassified = UIColor(light: Asset.Colors.blue100Light, dark: Asset.Colors.blue700Light)
```
Comment (KaterinaWire) [inline Wire-iOS/Sources/Helpers/SemanticColors.swift:134]: Same here, double check with Anette

File: Wire-iOS/Sources/UserInterface/Calling/BottomSheetContainerViewController/CallingBottomSheetViewController.swift [hunk 3]
```diff
@@ -36,14 +36,21 @@ class CallingBottomSheetViewController: BottomSheetContainerViewController {
...
         switch voiceChannel.state {
         case .incoming:
-            return UIDevice.current.orientation.isLandscape ? 128.0 : 230.0
+            offset = UIDevice.current.orientation.isLandscape ? 128.0 : 230.0
         default:
-            return 128.0
+            offset = UIDevice.current.orientation.isLandscape ? 128.0 : 112.0
         }
+        if case .established(_) = callInfoConfiguration?.state, let configuration = callInfoConfiguration, configuration.classification != .none {
+            offset += SecurityLevelView.SecurityLevelViewHeight
```
Comment (KaterinaWire) [inline Wire-iOS/Sources/UserInterface/Calling/BottomSheetContainerViewController/CallingBottomSheetViewController.swift:51]: Swift lint issue

File: Wire-iOS/Sources/UserInterface/Calling/BottomSheetContainerViewController/CallingBottomSheetViewController.swift [hunk 4]
```diff
@@ -163,18 +172,43 @@ class CallingBottomSheetViewController: BottomSheetContainerViewController {
         visibleVoiceChannelViewController.configurationObserver = self
         visibleVoiceChannelViewController.delegate = self
         callingActionsInfoViewController.setCallingActionsViewDelegate(actionsDelegate: visibleVoiceChannelViewController)
-        headerBar.setTitle(title: voiceChannel.conversation?.displayName ?? "")
+//        headerBar.setTitle(title: voiceChannel.conversation?.displayName ?? "")
```
Comment (KaterinaWire) [inline Wire-iOS/Sources/UserInterface/Calling/BottomSheetContainerViewController/CallingBottomSheetViewController.swift:175]: old code?

File: Wire-iOS/Sources/UserInterface/Calling/BottomSheetContainerViewController/CallingBottomSheetViewController.swift [hunk 5]
```diff
@@ -163,18 +172,43 @@ class CallingBottomSheetViewController: BottomSheetContainerViewController {
...
         callingActionsInfoViewController.participants = voiceChannel.getParticipantsList()
         participantsObserverToken = voiceChannel.addParticipantObserver(self)
     }
 
     override func bottomSheetChangedOffset(fullHeightPercentage: CGFloat) {
         overlay.alpha = fullHeightPercentage * 0.7
     }
+
+    //----------------------------------------------
```
Comment (KaterinaWire) [inline Wire-iOS/Sources/UserInterface/Calling/BottomSheetContainerViewController/CallingBottomSheetViewController.swift:184]: do we need this?

File: Wire-iOS/Sources/UserInterface/Calling/CallInfoViewController/BitRateLabel.swift [hunk 6]
```diff
@@ -18,13 +18,15 @@
...
+import WireCommonComponents
 
 enum BitRateStatus: String {
     case constant
     case variable
 
     fileprivate var localizedUppercasedText: String {
-        return "call.status.\(rawValue)_bitrate".localized(uppercased: true)
+        let uppercased = !DeveloperFlag.isUpdatedCallingUI
+        return "call.status.\(rawValue)_bitrate".localized(uppercased: uppercased)
```
Comment (KaterinaWire) [inline Wire-iOS/Sources/UserInterface/Calling/CallInfoViewController/BitRateLabel.swift:29]: For new UI we should use `.capitalized`, right?

Comment (MarcinR) [reply Wire-iOS/Sources/UserInterface/Calling/CallInfoViewController/BitRateLabel.swift:29]: no, it shouldn't but theres negation in line 28s condition

**Pull request closed without merging**